src/org/usfirst/frc4904/robot/commands/IntakeRollersIntake.java

```diff
@@ -0,0 +1,27 @@
+package org.usfirst.frc4904.robot.commands;
+
+
+import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.robot.subsystems.CrateIO;
+import org.usfirst.frc4904.standard.LogKitten;
+import org.usfirst.frc4904.standard.commands.KittenCommand;
+import org.usfirst.frc4904.standard.commands.motor.MotorConstant;
+import edu.wpi.first.wpilibj.command.CommandGroup;
+
+/**
+ * This command allows the rollers in
+ * CrateIO to spin at a constant rate
+ * towards the robot, allowing it to
+ * intake cubes that are in front of it.
+ */
+public class IntakeRollersIntake extends CommandGroup {
+	public IntakeRollersIntake() {
+		super("IntakeRollersIntake");
+		requires(RobotMap.Component.crateIO);
+		// requires(RobotMap.Component.crateIO.rollerLeft);
+		// requires(RobotMap.Component.crateIO.rollerRight);
+		addParallel(new KittenCommand("We is intaking boios", LogKitten.KittenLevel.WTF));
+		addParallel(new MotorConstant(RobotMap.Component.crateIO.rollerLeft, CrateIO.INTAKE_SPEED));
+		addParallel(new MotorConstant(RobotMap.Component.crateIO.rollerRight, CrateIO.INTAKE_SPEED));
```

Comment:
Stop using both of these seperately, instead use the `set` command included in teh subsystem